web-bluetooth/write-descriptor-async-await.js

```diff
@@ -0,0 +1,61 @@
+var myDescriptor;
+
+async function onReadButtonClick() {
+  let serviceUuid = document.querySelector('#service').value;
+  if (serviceUuid.startsWith('0x')) {
+    serviceUuid = parseInt(serviceUuid);
+  }
+
+  let characteristicUuid = document.querySelector('#characteristic').value;
+  if (characteristicUuid.startsWith('0x')) {
+    characteristicUuid = parseInt(characteristicUuid);
+  }
+
+  try {
+    log('Requesting any Bluetooth Device...');
+    const device = await navigator.bluetooth.requestDevice({
+     // filters: [...] <- Prefer filters to save energy & show relevant devices.
```

Comment:
I'm not sure we should acceptAllDevices in this sample. Depends on if devices advertise services that have this descriptor type. 

I do think we should suggest a few example services that have characteristics with this descriptor. Otherwise it may be unlikely for anyone to run this and have any success. Or, maybe we should really strongly encourage the use of BLE Sim app?